### ISSUE TITLE
Tasks won't consume by workers when scaling in kubernetes

### ISSUE THREAD

**Issue by mohammadrezza** [RichardKnop/machinery#578]
I am developing a service that uses machinery for task queue with redis. When I scale my service into 2 or more instances with the same default  queue, workers won't do the jobs. I fixed this problem with adding a [short id](https://github.com/teris-io/shortid) to default queue after reading config and before starting a new server for every instance of my service when they are initializing machinery. In this way all my services have their own default queue. Is this the right way to do this? If so can I create a pull request for adding a  _**Scale**_ option for machinery ?

**Comment by dxsooo** [comment]
maybe same problem with #559 ? You can boot up 2 worker and insert many tasks in queue to check this

**Comment by yap-studios** [comment]
@mohammadrezza, I'm using kubernetes and I can scale quite fine without the workaround you mentioned. Can you post a snippet of how you spin up your machinery service?
